### DISCUSSION TITLE
Resizable chat box

### DISCUSSION THREAD

**Discussion by expert3vision** [night/betterttv#5823] [Ideas]
Make the chat box resizable horizontally, so that longer mesages can fit on one line. This can be especially useful on ultrawide monitors when maximized.

**Comment by night** [comment]
Thanks for the suggestion. We used to actually have this feature long ago, but had to remove it due to complexities around Twitch's channel page resize handling. It's basically unfeasible to resize the chat and have the page render/resize properly due to computed dimensions that Twitch plumbs into the DOM.
